data/utils/zone_around_point.py

```diff
@@ -0,0 +1,58 @@
+"""
+Another method with geodesic to have exactly radius_m meters around a port, no matter where on the globe
+(polygons of the ports in the north seams to be flat, it's normal, it's the projection)
+"""
+
+import os
+import geopandas as gpd
+import pandas as pd
+import pyproj
+from shapely import wkt
+from shapely.geometry import Polygon
+
+radius_m = 3000  # Radius in kilometers
```

Comment:
C'est pas un peu beaucoup 3000km ?

Reply:
C'est en metres je me suis trompé dans le commentaire. Le code est bon

Fix: radius_m = 3000  # Radius in meters

Reply:
Tu peux corriger ?